packages/components/src/custom-select-control-v2/styles.ts

```diff
@@ -133,7 +134,16 @@ export const SelectItem = styled( Ariakit.SelectItem )`
 	justify-content: space-between;
 	padding: ${ ITEM_PADDING };
 	font-size: ${ CONFIG.fontSize };
-	line-height: 2.15rem; // TODO: Remove this in default but keep for back-compat in legacy
+	// Legacy line height is 28px.
+	// TODO: reassess for non-legacy v2
+	line-height: calc( 28px / ${ CONFIG.fontSize } );
+	scroll-margin: ${ space( 1 ) };
+	user-select: none;
+
+	&[aria-disabled='true'] {
+		opacity: 0.5;
+	}
```

Comment:
I actually omitted the disabled styles on purpose, because the item styles are fully customizable and the appropriate disabled style will vary. For example, adding a `disabled` to the "Purple" item in the [Default story](https://wordpress.github.io/gutenberg/?path=/story/components-customselectcontrol-v2-default--default) doesn't result in an understandable visual.

https://github.com/WordPress/gutenberg/blob/9a331f13e8e1c59b9defb27a0bae38265de38578/packages/components/src/custom-select-control-v2/types.ts#L196-L202

What do you think?